tests/SiteTreeSubsitesTest.php

```diff
@@ -8,6 +8,10 @@ class SiteTreeSubsitesTest extends BaseSubsiteTest {
 		'SiteTreeSubsitesTest_ClassA',
 		'SiteTreeSubsitesTest_ClassB'
 	);
+
+	protected $illegalExtensions = array(
```

Comment:
This was added to simplify testing of this module with translatable included.

Possibly we could add subsites + translatable specific tests in the future.